docs/end-user-flows/account-settings/by-management-api.mdx

```diff
@@ -0,0 +1,183 @@
+
+## Architecture
+
+```mermaid
+  graph TB
+    A[User] --> B[Client]
+    B -->|Account settings API call|C[Server-side application]
+    C -->|Management API call| D[Logto]
+```
+
+1. **User**: Authenticated end-user who needs to access and manage their account settings.
+2. **Client**: Your client application that serves the account settings page to the user.
+3. **Server-side application**: Server-side application that provides the account settings API to the client. Interacts with the Logto **Management API**.
+4. **Logto**: Logto as the authentication and authorization service. Provides the **Management API** to manage user accounts.
+
+## Steps to implement
+
+```mermaid
+  sequenceDiagram
+    autonumber
+    actor User as User
+    participant Client as Client app
+    participant Server as Server-side application
+    participant Logto as Logto
+
+    User ->> Client: Access client app
+    Client ->> Logto: POST /oidc/auth
+    User -->> Logto: sign in
+    Logto -->> Client: Redirect to client app
+    Client ->> Logto: POST /oidc/token
+    Logto ->> Client: Access token A
+    Client ->> Server: GET /account-settings (with access token A)
+    Server ->> Logto: POST /oidc/token (with client credentials)
+    Logto ->> Server: Access token B
+    Server ->> Logto: GET /api/users/{userId} (with access token B)
+    Logto ->> Server: User details
+    Server ->> Client: User details
+```
+
+1. User accesses the client application.
+2. Client application send the authentication request to Logto and redirects the user to the Logto sign-in page.
+3. User signs in to Logto.
+4. Authenticated user is redirected back to the client application with the authorization code.
+5. Client application requests the access token from Logto for the self-hosted account settings API access.
+6. Logto grants the access token to the client application.
+7. The client application send the account settings request to Server-side application with the user access token.
+8. Server-side application verifies the requester's identity and permission from the user access token. Then request for a management API access token from Logto.
+9. Logto grants the management API access token to Server-side application.
+10. Server-side application requests the user data from Logto using the management API access token.
+11. Logto verifies the server's identity and management API permission and returns the user data.
+12. Server-side application process the user data based on the requester's permission and returns the user account details to the client application.
+
+## User management APIs
+
+### Integrate management APIs
+
+Check the [management API](#) section to learn how to integrate the management APIs with Server-side application-side application.
+
+### User data schema
+
+Check the [User data and custom data](#) section to learn more about the user schema in Logto.
```

Comment:
nit: Add a "TODO" comment in case you forget to update the link

Reply:
I'll query the # gloably just like TODO

Reply:
OK